public/css/timeline.less

```diff
@@ -97,6 +112,22 @@
     &.sortable-ghost {
       opacity: .5;
     }
+
+    > a {
+      text-decoration: none;
+
+      i {
+        display: none;
+      }
+
+      &:hover {
+        color: @list-item-title-hover-color;
+
+        i {
+          display: inline-block;
+        }
+      }
```

Comment:
The fact that `i` uses `inline-block` by default, is only the case if it is also an `.icon`. This isn't addressed here.

I'd suggest to do this instead:

```
&:not(:hover) .icon {
  display: none;
}
```

This way, it resets to the default display value without the need to repeat it here. And it doesn't rely on the fact that an `ipl\Web\Icon` uses `i` as tag.